lib/linux_admin/ip_addr.rb

```diff
@@ -0,0 +1,31 @@
+module LinuxAdmin
+  class IpAddr
```

Comment:
I would not name this so similarly to the Ruby class.  Maybe just spell it out (`IpAddress`)?  Should it be part of a larger class like `NetworkConfig` or something?